ush/extractvars_tools.sh

```diff
@@ -48,8 +43,12 @@ copy_to_comout() {
   rundir_outfile=$1 #output data file generated in RUNDIR
   comout_dir=$2 #destination directory to which to copy the data file
   if [[ -f "${rundir_outfile}" ]];then
-    cp -pr "${rundir_outfile}" "${comout_dir}"
+    cpfs "${rundir_outfile}" "${comout_dir}"
   else
-   echo "WARNING: Output variable (${rundir_outfile}) does not exist."
+    echo "WARNING: Output variable (${rundir_outfile}) does not exist."
```

Comment:
Should this be a fatal error?

Reply:
If an output file for a given lead time has failed to produce, I do not think that should cause the entire rocoto task to fail, especially since all lead times (and ocean, ice, atmos and wave compoents) are being processed in a single rocoto task.

Reply:
@WalterKolczynski-NOAA If an output file for a given lead time has failed to produce, I do not think that should cause the entire rocoto task to fail, especially since all lead times (and ocean, ice, atmos and wave compoents) are being processed in a single rocoto task.

Reply:
The issue is if you don't fail and exit with non-zero, the rocoto task is going to show as `SUCCEEDED` and things will continue merrily on their way. The only way you'll realize you're missing data is looking at the logs (or the data) carefully.

Reply:
I agree that the rocoto task should not be displayed as "SUCCEEDED" if a file copy is unsuccessful. Is there an option to make the task continue if there is an error, but have rocoto display this task as "FAILED" after the entire task is finished? If not, I can make this error fatal and have the task end when this error occurs (this is more strict than what we did with the GEFSv12 reforecast, but I think we can make it work). If a copy failure does occur for just one lead time, we may need to rerun the entire task and regenerate all the final products anyway.

Reply:
The only way would be to save it somewhere and use it at the end. But I think it is safe to just fail immediately, as if it is missing data you are likely going to want to rerun it (and likely the whole cycle) again anyway.

Reply:
Ok. I have added a FATAL ERROR if `rundir_outfile` does not exist.